Mage/src/main/java/mage/cards/Card.java

```diff
@@ -40,12 +41,21 @@
 import mage.constants.Zone;
 import mage.counters.Counter;
 import mage.counters.Counters;
+import mage.filter.FilterMana;
 import mage.game.Game;
 import mage.game.GameState;
 import mage.game.permanent.Permanent;
 
 public interface Card extends MageObject {
 
+
+    final String regexBlack = ".*\\x7b.{0,2}B.{0,2}\\x7d.*";
```

Comment:
because it a interfce it's static by default but can the keyword be added to make it clear?